### PR TITLE
Add VideoNG as a project

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

**Review by dalecurtis — APPROVED** [review]

**Review by dalecurtis — APPROVED** [review]
still lgtm

**Comment by chrishtr** [comment]
The video team approved, let's commit this one!

**Comment by chrishtr** [comment]
@rachelandrew can you review?

File: site/en/blog/renderingng/index.md [hunk 1]
```diff
@@ -541,6 +541,53 @@ It began in 2015 and will finish in 2021.
...
+      <td>Shipped L1/4K protected content rendering support on ChromeOS.</td>
+    </tr>
+  </tbody>
+</table>
+
+Definitions of terms in the chart above:
+
+Mojo : A next-generation IPC subsystem for Chromium.
+
+Surface : A concept that is part of the Viz project design.
```
Comment (rachelandrew) [inline site/en/blog/renderingng/index.md:589]:
```suggestion
Mojo
: A next-generation IPC subsystem for Chromium.

Surface 
: A concept that is part of the Viz project design.
```

Comment (rachelandrew) [reply site/en/blog/renderingng/index.md:589]: makes these a definition list like the others.

File: site/en/blog/renderingng/index.md [hunk 2]
```diff
@@ -541,6 +541,53 @@ It began in 2015 and will finish in 2021.
...
+    <tr>
+      <td>2018</td>
+      <td>Shipped Mojo-based video decoding pipeline.</td>
+    </tr>
+    <tr>
+      <td>2019</td>
+      <td>Shipped Surface-based video rendering pipeline.</td>
+    </tr>
+      <td>2021</td>
+      <td>Shipped L1/4K protected content rendering support on ChromeOS.</td>
```
Comment (rachelandrew) [inline site/en/blog/renderingng/index.md:580]: Could we add a definition of L1/4K in the list below?

Comment (chrishtr) [reply site/en/blog/renderingng/index.md]: Done with https://github.com/chrishtr/developer.chrome.com/commit/16a6f01d49cc4704232525394df3f46ce57cc859.

I'm waiting for final verification from @dalecurtis.

**Comment by chrishtr** [comment]
Dale approved the latest edits (I removed the word L1). Ready to commit from my POV.

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**